example/color/FAO_with_color_example.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+from raceplotly.plots import barplot
+
+
+data = pd.read_csv('https://raw.githubusercontent.com/lc5415/raceplotly/main/example/FAOSTAT_data.csv')
+
+# To add specific color to the categories, a new coloumn with rgb values for each category has to be created.
+# Assigning colors to the categories.
+colors = {'Sugar cane': 'rgba(0, 76, 109, 1)',
```

Comment:
I've added a test for this, that is always good practice!

Reply:
Thanks a lot. I will take care of it the next time